powerdns/models/templates.py

```diff
@@ -115,3 +124,14 @@ def update_templated_records(sender, instance, **kwargs):
         pk__in=existing_template_ids,
     ):
         template.create_record(instance)
+
+
+@receiver(
+    post_save,
+    sender=RecordTemplate,
+    dispatch_uid='record_template_modify_templated_records',
+)
+def modify_templated_records(sender, instance, **kwargs):
+    for record in instance.record_set.all():
```

Comment:
Are records that were created automatically and later modified manually should be overwritten?